pkg/adapter/adapter.go

```diff
@@ -107,25 +108,27 @@ func (s *sinker) getHTTPClient() (*http.Client, error) {
 	var (
 		caCert    []byte
 		tlsConfig *tls.Config
+		count     int
 	)
 
 	certPool := x509.NewCertPool()
 	err := wait.PollImmediate(interval, timeout, func() (bool, error) {
-		clusterInterceptorList, err := clusterinterceptorsinformer.Get(s.injCtx).Lister().List(labels.NewSelector())
+		clusterInterceptorList, err := s.Clients.TriggersClient.TriggersV1alpha1().ClusterInterceptors().List(s.injCtx, metav1.ListOptions{
```

Comment:
Could we use a lister here instead of making a direct API call - this might fix the issue described in https://github.com/tektoncd/triggers/issues/1374#issuecomment-1174421352 and https://github.com/tektoncd/triggers/issues/1284#issuecomment-1173683986

Reply:
Latest code have lister only https://github.com/tektoncd/triggers/blob/main/pkg/adapter/adapter.go#L114 but still users getting issue related to timeout 
Do we need to check some other places :thinking:

Reply:
Updated to use listers